yaml_expander/yaml_expander.py

```diff
@@ -96,7 +96,7 @@ def expand(self, var):
                     env.append(c)
                     while c is not None:
                         c = chars.get()
-                        if c is not None and c.isalnum():
+                        if c is not None and (c.isalnum() or c in "-_"):
```

Comment:
Jeg er ikke vild med at $solr-doc-store-mappings ikke er klart hvor den deler... jeg kan gå med til - i ${}, men andensteds vil jeg stritte i mod.

Reply:
Det kan vi godt blive enige om.
-_ kun inden for {} par